backtrace-library/src/main/java/backtraceio/library/interfaces/Breadcrumbs.java

```diff
@@ -0,0 +1,10 @@
+package backtraceio.library.interfaces;
+
+import java.util.Map;
+
+import backtraceio.library.enums.BacktraceBreadcrumbLevel;
+import backtraceio.library.enums.BacktraceBreadcrumbType;
+
+public interface Breadcrumbs {
```

Comment:
`BacktraceBreadcrumbs` class that implements this interface has more than one public method. I think we should extend this insterface with other methods that belongs here - like for example enabling breadcrumbs:
```
  public boolean enableBreadcrumbs(Set<BacktraceBreadcrumbType> enabledBreadcrumbTypes,
                                  boolean isNewInstance,
                                  BacktraceBase staleBreadcrumbLogSender)
```

Reply:
I think other implementations should be able to enableBreadcrumbs as long as they want. 
I think this interface is important just to standardize `addBreadcrumb` method since we use this throughout the code base. Other methods are really just supporting `addBreadcrumb`. 

For instance now in our `backtrace-android.h` header we can accept any object which implements `addBreadcrumb` interface when we initialize.

Reply:
I have extended the `Breadcrumbs` interface to contain more methods, thank you!

Reply:
> I think other implementations should be able to enableBreadcrumbs as long as they want.
I think this interface is important just to standardize addBreadcrumb method since we use this throughout the code base. Other methods are really just supporting addBreadcrumb.

yeah, but on the other side you're expecting an implementation of `getBreadcrumbLogPath` `enableBreadcrumbs` etc methods in the backtraceDatabase implementation. WIth this approach user won't be able to inject his own implementation of breadcrumbs class.

Thanks for extending breadcrumbs interface with those methods!